test/functional/test_framework/wallet.py

```diff
@@ -182,28 +182,30 @@ def sendrawtransaction(self, *, from_node, tx_hex):
         from_node.sendrawtransaction(tx_hex)
         self.scan_tx(from_node.decoderawtransaction(tx_hex))
 
-def make_chain(node, address, privkeys, parent_txid, parent_value, n=0, parent_locking_script=None, fee=DEFAULT_FEE):
-    """Build a transaction that spends parent_txid.vout[n] and produces one output with
+def make_chain(node, address, privkeys, parent_txid, parent_value, parent_locking_script=None, fee=DEFAULT_FEE):
```

Comment:
https://github.com/bitcoin/bitcoin/pull/21800/files#r685091478 also suggested renaming this function

I'd suggest something like `create_child` ?